exporter/prometheusexporter/accumulator.go

```diff
@@ -256,13 +256,13 @@ func (a *lastValueAccumulator) Collect() []pdata.Metric {
 	return res
 }
 
-func timeseriesSignature(ilmName string, metric pdata.Metric, labels pdata.StringMap) string {
+func timeseriesSignature(ilmName string, metric pdata.Metric, attributes pdata.AttributeMap) string {
 	var b strings.Builder
 	b.WriteString(metric.DataType().String())
 	b.WriteString("*" + ilmName)
 	b.WriteString("*" + metric.Name())
-	labels.Sort().Range(func(k string, v string) bool {
-		b.WriteString("*" + k + "*" + v)
+	attributes.Sort().Range(func(k string, v pdata.AttributeValue) bool {
+		b.WriteString("*" + k + "*" + v.StringVal()) // TODO (codeboten): Decide what to do w/ non-string values
```

Comment:
File an issue? Or add this to the 0.9.0 upgrade issue?

Reply:
Added all the code that contained TODOs in the issue https://github.com/open-telemetry/opentelemetry-collector/issues/3815, which has been added to the checklist here: https://github.com/open-telemetry/opentelemetry-collector/pull/3740

Might be worth updating the checklist for #3535 instead.